### PR TITLE
Set difference1d tensor

### PR DESCRIPTION
**Pull request by rishabh-mondal**
Return the elements in `a` that are not in `b'

a = torch.tensor([1, 2, 3, 4, 5])
b = torch.tensor([3, 4, 6, 7, 8])

result =([1,2,5])
mask = ~a.unsqueeze(1).eq(b).any(dim=1)
Creates a Boolean mask to identify elements in a that are not present in b.

torch.masked_select(a, mask)
torch.masked_select function to select elements from tensor 'a' based on the Boolean mask.

### CODE REVIEW

**Comment by patel-zeel** [comment]
This PR should change only `utils.py`. Currently, it changes 3 files in total.

**Comment by rishabh-mondal** [comment]
PR is updated please check.

**Comment by patel-zeel** [comment]
@rishabh-mondal Rename the function to exactly match with numpy function. Add tests where you use numpy inbuilt function and verify that your function produces the same output. Please add runtime information in this PR as well.

**Comment by patel-zeel** [comment]
@rishabh-mondal, can you please do what is needed for this PR?
